styles/hud/ground-boost.scss

```diff
@@ -21,18 +22,19 @@
 		height: 100%;
 		width: 100%;
 
-		&--knockback {
+		&--friction {
 			wash-color: $red;
 		}
 
-		&--groundboost {
+		&--slick {
 			wash-color: $blue;
 		}
 	}
 
 	&__background--meter {
```

Comment:
Sorry, I must have missed this in previous review. Why are you using two hyphens here, surely `--meter` isn't a modifier? Don't you want `background-meter`?

Reply:
good catch. Definitely should have been `background-meter`. Fixed